test/test_assets.js

```diff
@@ -295,7 +295,7 @@ describe("assets", function () {
           "$uri: 'assets/foo/bar.png');" +
           ".test { foo: inspect($eg-registered-assets); }";
```

Comment:
change this line to `.test { foo: inspect(map-get(map-get($eg-registered-assets, module-a), 'foo/bar.png')); }` and adjust the output accordingly.

Reply:
```
¦.test {
  ¦ ¦ -  foo: (module-a: ("foo/bar.png": (filepath: "images/foo/bar.png", uri: "assets/foo/bar.png"))); }
  ¦ ¦ +  foo: (filepath: "images/foo/bar.png", uri: "assets/foo/bar.png"); }
```

Reply:
the new output looks good, we can set the expected value to that and the tests should pass now, yes?